src/plugins/irc-events/message.js

```diff
@@ -10,6 +10,11 @@ module.exports = function(irc, network) {
 			return;
 		}
 
+		if (data.message.substring(0,4) == '?OTR') {
+			// Hide OTR messages
+			return;
+		}
```

Comment:
btw, hating single-line without curly braces, makes code less readable and not diff-friendly when evolving